packages/ui/src/2_molecules/VerticalTabs/components/VerticalTabItem.tsx

```diff
@@ -22,8 +23,12 @@ export const VerticalTabItem: FC<VerticalTabsItemButtonProps> = ({
       data-active={active}
       onClick={onClick}
     >
-      <p className={styles.label}>{label}</p>
-      {infoText && <small className={styles.info}>{infoText}</small>}
+      <Paragraph size={ParagraphSize.base}>{label}</Paragraph>
+      {infoText && (
+        <Paragraph size={ParagraphSize.small} className={styles.info}>
```

Comment:
```suggestion
        <Paragraph className={styles.info}>
```
`ParagraphSize.small` is default `size`.